image/verify/src/verifier.rs

```diff
@@ -586,7 +587,17 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
     // Check if SVN check is required
     fn svn_check_required(&mut self) -> bool {
```

Comment:
always(inline) to ensure that the cfi checks below are included in every check? Otherwise the code that checks the return value of this function is vulnerable to glitching...

Reply:
Updated